Our base does not have libutempter, so no KPty_HAVE_UTEMPTER
Port is build as binary executable.

--- pkg-plist.orig	2018-09-10 20:50:59 UTC
+++ pkg-plist
@@ -1,2 +1,3 @@
-%%QT_PLUGINDIR%%/kf5/kded/kwrited.so
+bin/kwrited
+etc/xdg/autostart/kwrited-autostart.desktop
 share/knotifications5/kwrited.notifyrc
